sql/src/test/resources/drill/window/queries/druid_queries/partition_by_array/wikipedia_query_1.q

```diff
@@ -0,0 +1,6 @@
+select
+countryName, cityName, channel,
+row_number() over (partition by array[1,2,length(cityName)] order by countryName) as c
```

Comment:
`partition by array[1,2,length(cityName)]` is same as `partition by length(cityName)`.
To provide the difference may be can we try out `partition by array[1, expr1, expr2]`?

Reply:
I really like this suggestion!
Have added such a test, thanks a lot for pointing this out! :D